### PR TITLE
N°6173 \HTMLSanitizer::Sanitize : Fix handling only svg_sanitizer

### PR DESCRIPTION
**Pull request by piRGoif**
As suggested by Hipska in https://github.com/Combodo/iTop/pull/449#discussion_r1118397193

### CODE REVIEW

File: core/htmlsanitizer.class.inc.php [hunk 1]
```diff
@@ -51,8 +51,7 @@ public static function Sanitize($sHTML, $sConfigKey = 'html_sanitizer')
 			if ($sConfigKey === 'html_sanitizer') {
 				IssueLog::Warning('The configured "'.$sConfigKey.'" class "'.$sSanitizerClass.'" is not a subclass of HTMLSanitizer. Will use HTMLDOMSanitizer as the default sanitizer.');
```
Comment (Hipska) [inline core/htmlsanitizer.class.inc.php]:
```suggestion
				IssueLog::Warning('The configured "html_sanitizer" class "'.$sSanitizerClass.'" is not a subclass of '.HTMLSanitizer::class.'. Will use HTMLDOMSanitizer as the default sanitizer.');
```

**Review by Hipska — APPROVED** [review]

File: core/htmlsanitizer.class.inc.php (hunk 1 above)
Comment (piRGoif) [reply core/htmlsanitizer.class.inc.php]: I like the second suggestion, but prefer not to replace $sConfigKey with its literal value.
Pushed in 0c115c28

Comment (Hipska) [reply core/htmlsanitizer.class.inc.php]: It can't be any other value and it makes searching for errors in the source code easier.

**Review by Hipska — APPROVED** [review]

**Review by Molkobain — APPROVED** [review]

File: core/htmlsanitizer.class.inc.php (hunk 1 above)
Comment (piRGoif) [reply core/htmlsanitizer.class.inc.php]: We tend to limit as much as possible literals in the code. If 'html_sanitizer' is renamed, we would have to do the rename twice here...

**Comment by Molkobain** [comment]
Accepted during technical review

**Comment by piRGoif** [comment]
Accepted in functional review.
SVG sanitizer was introduced in 2.6.5 / 2.7.6 / 3.0.0 so ok to introduce fix in support/2.7

**Pull request merged**